src/model/schema.js

```diff
@@ -1285,6 +1344,15 @@ export class SchemaContext {
  * @typedef {Object} module:engine/model/schema~SchemaContextItem
  */
 
+/**
+ * A structure containing additional metadata describing the attribute.
```

Comment:
As I wrote earlier, `setAttributeProperties()` allows any property names while this type clearly narrows possible options to `isFormatting` only. We should figure that out.